commit 3584c82e05d49cf4638dd52f038530ff6c411b41 (refs/changes/61/3861/3)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-06-26T11:36:28+00:00 (1 year, 3 months ago)
    
    TRITON-324 node-triton cli-affinity.test.js failures: create timeout, .end() called twice
    Reviewed by: Pedro P. Candel <pedro@joyent.com>
    Approved by: Pedro P. Candel <pedro@joyent.com>

diff --git a/test/integration/cli-affinity.test.js b/test/integration/cli-affinity.test.js
index f427aa6..d68cca3 100644
--- a/test/integration/cli-affinity.test.js
+++ b/test/integration/cli-affinity.test.js
@@ -110,7 +110,8 @@ test('affinity (triton create -a RULE ...)', testOpts, function (tt) {
     var db2Alias = ALIAS_PREFIX + '-db2';
     var db2;
     tt.test('  triton create -n db2 -a \'instance!=db*\'', function (t) {
-        var argv = ['create', '-wj', '-n', db2Alias, '-a', 'instance!=db*',
+        var argv = ['create', '-wj', '-n', db2Alias, '-a',
+            'instance!=' + ALIAS_PREFIX + '-db*',
             imgId, pkgId];
         h.safeTriton(t, argv, function (err, stdout) {
             var lines = h.jsonStreamParse(stdout);
